lib/Rex/Logger.pm

```diff
@@ -171,6 +174,16 @@ sub info {
   }
 }
 
+sub log_error { logger( shift, "error" ) }
+sub log_warn  { logger( shift, "warn"  ) }
+sub log_info  { logger( shift, "info"  ) }
+sub log_debug { debug(@_) }
+
+sub error { logger( shift, "error" ) }
+sub warn  { logger( shift, "warn"  ) }
+sub info  { logger( shift, "info"  ) }
```

Comment:
I'm afraid this breaks existing `Rex::Logger::info` calls in the wild, at least when they specified `warn` or `error` levels :/ Am I missing something? Maybe I'm mistaken, didn't test it yet.